Xamarin.Forms.Core/ActivityIndicator.cs

```diff
@@ -1,5 +1,9 @@
 using System;
+using Xamarin.Forms;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-activityindicator-color", typeof(ActivityIndicator), nameof(ActivityIndicator.ColorProperty))]
```

Comment:
no need for a vendor-specific, use `color`

Reply:
Can we map one css property to multiple BPs? For `color`, `ActivityIndicator.ColorProperty` is not the same as `TextElement.TextColorProperty` to which `color` is currently bound.

I ran the possibility of augmenting the loader to support one to many bindings but he nixed that idea in favor of a convention that didn't require changing the loader.

Reply:
we're doing that elsewhere, by defining an interface and a static class, so the ColorProperty is actually a shared instance

Reply:
With one to many support this can now just be "color"